Geometry/GEMGeometry/src/ME0Chamber.cc

```diff
@@ -78,3 +78,23 @@ const ME0EtaPartition* ME0Chamber::etaPartition(int isl) const {
   }
   return 0;
 }
+
+float ME0Chamber::computeDeltaPhi(const LocalPoint& position, const LocalVector& direction ) const {
```

Comment:
@jshlee the reason why this is here and not in the ME0SegmentAlgorithm is that the same variable will have to be calculated for pixel tracks projected into the ME0Chamber. This allows for the variable to be obtained for both segments and tracks without code duplication.